### DISCUSSION TITLE
What should be the ideal number of nodes for quartz?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2275] [Q&A]
I want to enable the clustering in my service which is using quartz.net library.

I would like to know what is the ideal suggested number of nodes count?

I’m reading this Java article, it’s mentioned that 3 is the ideal number.

https://www.quartz-scheduler.org/documentation/quartz-2.3.0/configuration/ConfigJDBCJobStoreClustering.html#:~:text=Quartz%27s%20clustering%20features%20bring%20both,cluster%20share%20the%20same%20database.

is it true? Beyond that will it have negative impacts?

**Comment by viveknuna** [comment]
@lahma do you have any input on this?

**Comment by lahma** [comment]
The ideal number of nodes needs to come from your own data and analysis. It's impossible to say any number because it varies by use case.
